### PR TITLE
cleanup remote block headings

### PR DESCRIPTION
**Pull request by casperdcl**
Details blocks are `#id` linkable now (#3329) so should have [URL-friendly names](https://github.com/iterative/dvc.org/pull/3299#issuecomment-1048325250)

### CODE REVIEW

**Review by jorgeorpinel — APPROVED** [review]
Sure why not, looks better. Thanks!

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/get-url.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ $ dvc get-url webhdfs://user@example.com/path/to/file
 
 <details>
 
-### Click and expand for a local example
+### local
```
Comment (jorgeorpinel) [inline content/docs/command-reference/get-url.md:168]: Wait. Should be "Local"

File: content/docs/command-reference/remote/add.md [hunk 2]
```diff
@@ -364,7 +364,7 @@ $ dvc remote add -d myremote \
 
 <details>
 
-### Click for local remote
+### local remote
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/add.md:367]: Same

File: content/docs/user-guide/external-dependencies.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ $ dvc run -n download_file \
 
 <details>
 
-### Click for local file system paths
+### local file system paths
```
Comment (jorgeorpinel) [inline content/docs/user-guide/external-dependencies.md:136]: That's closer.

File: content/docs/user-guide/managing-external-data.md [hunk 4]
```diff
@@ -178,7 +178,7 @@ $ dvc run -d data.txt \
 
 <details>
 
-### Click for local file system paths
+### local file system paths
```
Comment (jorgeorpinel) [inline content/docs/user-guide/managing-external-data.md:181]: Same

**Comment by jorgeorpinel** [comment]
p.s. I'll fix it in #3388 ⌛

**Pull request merged**